browser-test/src/support/index.ts

```diff
@@ -319,10 +322,8 @@ export const setLangEsUS = async (page: Page) => {
  *     login can be initiated from different pages, for example after program
  *     submission.
  */
```

Comment:
There are no usages of `loginButton` below.

We also change it from `button` to `a` so that the test code always clicks on the "Log In" link on the header, rather than the "Log In" button on the programs page which is sometimes but not always available:

![image](https://user-images.githubusercontent.com/30369272/234912127-6f512e33-e7e3-4b16-a6ab-d93e1d3cd12b.png)